TOMP-API.yaml

```diff
@@ -2402,16 +2372,16 @@ components:
       schema:
         type: string
       description: Version of the API.
-      example: 1.1
+      example: 0.5.0
     maasId:
       in: header
       name: maas-id
```

Comment:
_maasId_

Reply:
kebab case => camelCase was only for the JSON.